rskj-core/src/test/java/co/rsk/peg/BridgeSupportTest.java

```diff
@@ -2644,7 +2705,7 @@ void when_registerBtcTransaction_usesLegacyType_beforeFork_lock_and_no_refund()
         provider.setNewFederation(federation1);
 
         // Whitelist the addresses
-        LockWhitelist whitelist = provider.getLockWhitelist();
+        LockWhitelist whitelist = whitelistStorageProvider.getLockWhitelist(activationsBeforeForks, btcRegTestParams);
```

Comment:
i think is better to use the `activations` that is part of this test

Reply:
Done!